app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -922,44 +948,47 @@ private void addNewMessage(String answer, boolean isMap, boolean isHavingLink, b
         } else {
             id = (long) temp + 1;
         }
-        if(answer.contains("google"))
-        {
+        if(answer!=null){
```

Comment:
@daminisatya Here I have added this condition so I provide some indentation down there